WeakAuras/RegionTypes/DynamicGroup.lua

```diff
@@ -931,34 +946,34 @@ local function modify(parent, region, data)
   function region:DoPositionChildren()
     WeakAuras.StartProfileSystem("dynamicgroup")
     WeakAuras.StartProfileAura(data.id)
-    if data and self.anchoredToUnit then
-      local anchors = {}
-      -- build a list keyed by anchor
-      for i, regionData in ipairs(self.sortedChildren) do
-        local anchor
-        local state = regionData.region.state
-        if state and state.unit then
-          if data.anchorFrameType == "NAMEPLATE" then
-            anchor = C_NamePlate.GetNamePlateForUnit(state.unit)
-          elseif data.anchorFrameType == "UNITFRAME" then
-            anchor = GetUnitFrame(state.unit)
+    if data and self.dynamicAnchor then
+      local frames = {}
+      if data.anchorFrameType == "CUSTOM" then
+        region.customAnchorFunc(frames, self.sortedChildren)
+      else
+        -- build a list keyed by frame
+        for i, regionData in ipairs(self.sortedChildren) do
+          local frame
+          local unit = regionData.region.state and regionData.region.state.unit
+          if unit then
```

Comment:
to call the customAnchorFunc here with either the regionData, or the state. I somewhat prefer the later as a parameter.

A third option would be that if a region.state has a "anchor" member, than that's the aura's anchor